_posts/2025-06-04-cockpit-340.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cockpit 340
+author: Venefilyn
+date: "2025-06-04"
+tags: cockpit, podman, files, machines, ostree
+slug: cockpit-340
+category: release
+summary: "Handle unsupported storage partitions, link quadlet logs to their service, source mode for virtual networks, various UX improvements."
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 340, cockpit-podman 107, cockpit-machines 333, and cockpit-ostree 211:
+
+## Storage: Prevent modifying partitions in unsupported places
+
+It was previously confusing and somewhat dangerous when storage page listed LVM2 Logical Volumes as having free space. This is now changed to include a notice and not display these partitions to prevent mishaps when we cannot support it.
```

Comment:
This is really *really* hard to understand. After talking to Freya a bit, this is the 50% that I understand:

```suggestion
It was previously confusing and somewhat dangerous when the Storage page listed LVM2 Logical Volumes as having free space. That is an internal implementation detail, and acting on that could have dangerous consequences. The page now does not display these partitions any more.
```

But honestly this sounds more like a bug fix, and just dropping that topic altogether would be less confusing altogether IMHO.